.vagrant/Vagrantfile

```diff
@@ -0,0 +1,196 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# NOTES:
+# This version was tested using Windows 11 Home Edition
+# It uses the "vagrant-gatling-rsync" Vagrant plugin to sync host folder to guest folder
+# It uses the "vagrant-rsync-back" plugin to sync guest folder to host folder
+# The sync from host to guest happens automatically on "vagrant up" after a "vagrant halt"
+# Once the sync is complete, rsync will monitor the host folder for changes and sync them to the guest folder.
+# To sync from guest to host, you must manually run "vagrant rsync-back" in a terminal window in vagrant project folder
+# Please install these plugins before issuing "vagrant up" by running the 2 commands in a terminal window.
+# vagrant plugin install vagrant-gatling-rsync
+# vagrant plugin install vagrant-rsync-back
+
+# Customize the following constants to suit your needs
+HOST_FOLDER = "../gloss-translation"
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "generic/ubuntu2204"
+
+  # Configure for application API
+  config.vm.network "forwarded_port", guest: 4300, host: 4300, host_ip: "127.0.0.1"
+
+  # Configure for application Web
+  config.vm.network "forwarded_port", guest: 4200, host: 4200, host_ip: "127.0.0.1"
+
+  # Mount repo project root folder.
+  config.vm.synced_folder HOST_FOLDER, "/home/vagrant/vmrepo", type: "rsync",
+    rsync__exclude: [".git/",".vagrant/","docs/", "node_modules/","packages/api/.env.local","packages/db/.env.local"]
```

Comment:
We will want to sync over the .env.local files so that the user can modify env vars. We can set the env vars we need to in this file, but we need to be able to set other secrets that we don't want to check into the public repo

Reply:
There is a problem if I remove `"packages/api/.env.local","packages/db/.env.local"` from the exclude list.
Namely, plugin `vagrant-gatling-rsync` is only a one-way sync from host to guest.  During the initial run of `vagrant up`, both .env.local files are created in guest environment only as there is no replication from guest to host.  When the inital `vagrant up` install is completed, Vagrant autostarts the  `vagrant-gatling-rsync` from host to guest. Consequently it deletes the 2 new .env.local files because they do not exist on the host side.

We can make the change you requested, however, as a work-around to this problem, we would have to update the user instructions to have them run the following 2 steps after the initial install shown below:  
The first line recreates both .env.local files. 
 The 2nd line syncs from guest to host.

```
vagrant provision --provision-with env
vagrant rsync-back
```

Reply:
Can you explain more the need for rsync? Doesn't vagrant sync folders out of the box without a plugin?

Reply:
Originally, I tried with default virtualbox syncing, but had issues with symbolic links. So this is why I switched to rsync. However, I retried default virtualbox at your request, and now it works fine.  Default is simpler and faster.  Also, default virtualbox works differently than rsync.  rsync does replication, where virutalbox performs a mount. As a result, you cannot exclude any directories with virtualbox.